packages/flutter/test/material/dialog_test.dart

```diff
@@ -2996,21 +3087,30 @@ class _RestorableDialogTestWidget extends StatelessWidget {
           },
           child: const Text('X'),
         ),
-      ),
+    ),
```

Comment:
Looks like an indent got removed randomly 🙃

```suggestion
      ),
```

Reply:
😂 fixed.